src/Associations.cc

```diff
@@ -537,127 +536,127 @@ void Associations::CollectMCStars(int realization)
+        refflux[8] = getflux(ds, "R");
+        refflux[9] = getflux(ds, "I");
 
-      RefStar* rs = new RefStar(ds, ds);
-      rs->AssignRefFluxes(refflux);
+        RefStar* rs = new RefStar(ds, ds);
+        rs->AssignRefFluxes(refflux);
 
-      unsigned int index = (unsigned int)ds.getval("index");
-      rs->Index() = index;
+        unsigned int index = (unsigned int)ds.getval("index");
+        rs->Index() = index;
 
-      photRefStarList.push_back(rs);
+        photRefStarList.push_back(rs);
     }
 
-  cout << photRefStarList.size() << " stars read" << endl;
+    cout << photRefStarList.size() << " stars read" << endl;
 
-  // Now, If I understand well, all the FittedStar coordinates
-  // are in CTP coordinates ==> we have to transform our
-  // (RA,DEC) coordinates into CTP coords...
-  GtransfoLin identity;
-  TanRaDec2Pix RaDec2CTP(identity, commonTangentPoint);
-  photRefStarList.ApplyTransfo(RaDec2CTP);
+    // Now, If I understand well, all the FittedStar coordinates
+    // are in CTP coordinates ==> we have to transform our
+    // (RA,DEC) coordinates into CTP coords...
+    GtransfoLin identity;
+    TanRaDec2Pix RaDec2CTP(identity, commonTangentPoint);
+    photRefStarList.ApplyTransfo(RaDec2CTP);
 
-  cout << "[DEBUG DEBUB DEBUG]" << endl;
-  RaDec2CTP.dump(cout);
-  cout << "[DEBUG DEBUB DEBUG]" << endl;
+    cout << "[DEBUG DEBUB DEBUG]" << endl;
+    RaDec2CTP.dump(cout);
+    cout << "[DEBUG DEBUB DEBUG]" << endl;
 }
 
 
 void Associations::AssociatePhotometricRefStars(double MatchCutInArcSec)
 {
-  cout << "[Associations::AssociatePhotometricRefStars]" << endl;
-  cout << "[DEBUG]  -> nb fs=" << fittedStarList.size()
-       << " nb refstars="  << photRefStarList.size() << endl;
+    cout << "[Associations::AssociatePhotometricRefStars]" << endl;
+    cout << "[DEBUG]  -> nb fs=" << fittedStarList.size()
+         << " nb refstars="  << photRefStarList.size() << endl;
 
-  // clear previous associations
-  RefStarIterator I;
-  for(I=photRefStarList.begin();I!=photRefStarList.end();I++)
+    // clear previous associations
+    RefStarIterator I;
+    for (I = photRefStarList.begin(); I != photRefStarList.end(); I++)
     {
-      (*I)->SetFittedStar( *(FittedStar*)NULL ); // mais c'est horrible (!)
-      //      cout << " XTP,YTP=" << (*I)->x << " " << (*I)->y << endl;
+        (*I)->SetFittedStar( *(FittedStar*)NULL ); // mais c'est horrible (!)
+        //      cout << " XTP,YTP=" << (*I)->x << " " << (*I)->y << endl;
     }
 
-  GtransfoIdentity gtransfoIdentity;
-  StarMatchList* smList = ListMatchCollect(Ref2Base(photRefStarList),
-					   Fitted2Base(fittedStarList),
-					   &gtransfoIdentity,
-					   MatchCutInArcSec/3600.);
+    GtransfoIdentity gtransfoIdentity;
+    StarMatchList* smList = ListMatchCollect(Ref2Base(photRefStarList),
+                            Fitted2Base(fittedStarList),
+                            &gtransfoIdentity,
+                            MatchCutInArcSec / 3600.);
```

Comment:
In fact, all of those `/3600` things should really be converted to use afw.angle. I'll file a ticket.